ext/standard/var_unserializer.c

```diff
@@ -137,6 +182,57 @@ static zval *var_access(php_unserialize_data_t *var_hashx, zend_long id)
 	return var_hash->data[id];
 }
 
+static int var_wakeup_all(php_unserialize_data_t *var_hashx, const zend_bool is_cleanup)
```

Comment:
Nothing else adds comments. I assume the conventions are 1 on success, 0 on failure, for all "boolean" functions.